Bluetooth: controller: Remove redundant BT_LL_SW_SPLIT conditional

cvinayak opened the pull request:
Remove redundant use of BT_LL_SW_SPLIT conditional.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

cvinayak committed:
Bluetooth: controller: Remove redundant BT_LL_SW_SPLIT conditional

Remove redundant use of BT_LL_SW_SPLIT conditional.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

kruithofa commented:
Please look at other places where the BT_LL_SW_SPLIT still is used, and whether that is correct or not

kruithofa reviewed (changes requested):
Please look at other places where the BT_LL_SW_SPLIT is used and see if this needs to be removed there as well

cvinayak committed:
tests: Bluetooth: shell: Remove cmd_ull_reset

Remove cmd_ull_reset command, as this would put the host out
of sync. Also, this command is seldom used/required.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

kruithofa reviewed (approved):
LGTM
